src/lang/modifyAst/sweeps.test.ts

```diff
@@ -513,3 +507,102 @@ revolve001 = revolve(profile001, angle = 10, axis = X)`
 )`)
   })
 })
+
+describe('Testing getAxisExpressionAndIndex', () => {
```

Comment:
Adding tests here for those helpers used by revolve and helix, first one in the codemod and second in the prepare to edit functions